Module Parser KETSU/wcos(1).txt

```diff
@@ -0,0 +1,675 @@
+{
+  "moduleInfo": {
+    "moduleName": "WCO",
+    "moduleInitials": "WCO",
+    "moduleDesc": "KETSU Module for wcostream.com",
+    "developer": "you're mom",
+    "moduleID": "25565",
+    "moduleImage": "https://www.wcostream.com/wp-content/themes/animewp78712/images/logo.gif",
+    "moduleVersion": 1.2,
+    "moduleLenguage": "ENG",
+    "moduleType": "Video",
+    "baseURL": "https://www.wcostream.com/",
+    "moduleDeveloperSite": "",
+    "UpdateSite": "",

```

Comment:
set the update site to the raw module json and the developer site to the library html or your own site.